systems/robotInterfaces/QPLocomotionPlanSettings.m

```diff
@@ -0,0 +1,366 @@
+classdef QPLocomotionPlanSettings
+  properties
+    robot;
+    contact_groups;
+    support_times
+    supports;
+    body_motions;
+    zmp_data;
+    zmptraj = [];
+    zmp_final = [];
+    LIP_height;
+    V;
+    qtraj;
+    comtraj = [];
+    mu = 0.5;
+    plan_shift = zeros(6,1);
+    plan_shift_zmp_inds = 1:2;
```

Comment:
these supercede (and replace) the options in the (now deprecated) qplocomotionplan.m